src/sgame/sg_buildable.cpp

```diff
@@ -1608,6 +1609,7 @@ static gentity_t *SpawnBuildable( gentity_t *builder, buildable_t buildable, con
 			break;
 
 		case BA_A_OVERMIND:
+			built->id = G_NewString( "overmind" );
```

Comment:
WDYT about leaving a bread crumb saying that this will be freed in in the enttiy cleanup function?